pkg/tags/tags.go

```diff
@@ -84,6 +84,7 @@ func ResolveLocal(ctx context.Context, c kclient.Client, namespace, image string
 		if IsLocalReference(image) {
 			return "", false, err
 		}
+		return image, false, nil
```

Comment:
@iwilltry42 is this accurate? you just want This single line change? This does not align with what was in your original change: https://github.com/acorn-io/acorn/pull/1682/files

Reply:
As I understand it, this is correct.

If the image is not found and not a local reference, then we essentially want to return "this image is not found and that's OK."

By not returning here, we'll go to the `else if !localImage.Remote` branch and essentially return `image, true, nil`.

Reply:
But if the first condition is true, won't it just jump to the end of the whole `if else if` chain and return `image, false, nil` anyway?

Reply:
Oh, geez. You're 100% correct. I had my nesting confused.

Reply:
Yeah I think that's correct too

Reply:
Huh, that's weird.
It looked different in the diff I first viewed.
It's only this one line but added to my previous change actually.
I just checked it again (on my phone right now) due to your comments and here I don't see the previous changes.
I'll fix it tomorrow morning.

Reply:
I guess it dropped my cherry-pick of the previous change.
It's a Git mystery.

Reply:
Replayed the revert of @cjellick's revert and rebased :heavy_check_mark: